### ISSUE TITLE
False positive for .tv TLD

### ISSUE THREAD

**Issue by benrugg** [ZooTools/email-spell-checker#26]
## False positive: anything@anything.tv suggests anything@anything.tk

1. How popular is the TLDs or Domain: `.tv` are used by many media companies and startups. Not sure about exact numbers, but I've seen several articles over the years about the [financial boon that .tv domain registration fees have been for the island of Tuvalu](https://theworld.org/stories/2022-01-24/tuvalu-cashes-its-coveted-internet-domain-name-amid-rise-online-streaming).
2. Why most of the companies using this package will get benefit of it: False positives for email address suggestions cause additional customer support costs.

## Current Behavior

```
const suggestion = emailSpellChecker.run({ email: 'anything@anything.tv' })
suggestion.full == 'anything@anything.tk'
```

## Expected Behavior

```
const suggestion = emailSpellChecker.run({ email: 'anything@anything.tv' })
suggestion == null
```

## Steps to Reproduce the Problem

1. See above
